charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -39,6 +39,35 @@ spec:
       serviceAccountName: machine-controller-manager
       terminationGracePeriodSeconds: 5
       containers:
+      - name: machine-controller-manager-provider-alicloud
+        image: {{ index .Values.images "machine-controller-manager-provider-alicloud" }}
+        imagePullPolicy: IfNotPresent
+        command:
+        - ./machine-controller
+        - --control-kubeconfig=inClusterConfig
+        - --target-kubeconfig=/var/lib/machine-controller-manager/kubeconfig
+        - --machine-creation-timeout=20m
+        - --machine-drain-timeout=2h
+        - --machine-health-timeout=10m
+        - --namespace={{ .Release.Namespace }}
+        - --port={{ .Values.metricsPortAlicloud }}
+        - --v=3
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: {{ .Values.metricsPortAlicloud }}
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /var/lib/machine-controller-manager
+          name: machine-controller-manager
+          readOnly: true
       - name: alicloud-machine-controller-manager
         image: {{ index .Values.images "machine-controller-manager" }}
         imagePullPolicy: IfNotPresent
```

Comment:
You missed adding this flag to the existing controller-manager deployment [like this](https://github.com/gardener/gardener-extension-provider-aws/pull/212/files#diff-e38adde4c814c39f0c3443b395a76dfc43639b003e8514c731b39e2b1683aea9R77).  It will help migration/deletion from both sides.

Reply:
done. https://github.com/gardener/gardener-extension-provider-alicloud/pull/224/files#diff-e38adde4c814c39f0c3443b395a76dfc43639b003e8514c731b39e2b1683aea9R77